Hanshul_bot/src/automation/googleAutomationFUN.java

```diff
@@ -0,0 +1,27 @@
+package automation;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class googleAutomationFUN {
+
+	public static void main(String[] args) {
+        System.setProperty("webdriver.chrome.driver", "/Users/deepti/Downloads/chromedriver");
+		WebDriver driver = new ChromeDriver();
```

Comment:
Please make enum to support multiple drivers

Reply:
hi

Reply:
public